data/json/vehicleparts/frames.json

```diff
@@ -56,10 +56,29 @@
       "removal": { "skills": [ [ "mechanics", 1 ] ], "time": "10 m", "using": "vehicle_weld_removal" },
       "repair": { "skills": [ [ "mechanics", 1 ] ], "time": "5 m", "using": [ [ "repair_welding_standard", 1 ] ] }
     },
-    "flags": [ "MOUNTABLE", "INITIAL_PART" ],
+    "flags": [ "MOUNTABLE", "INITIAL_PART", "NO_INSTALL_PLAYER" ],
     "categories": [ "hull" ],
     "damage_reduction": { "all": 20 }
   },
+  {
+    "id": "jrframe",
+    "type": "vehicle_part",
+    "name": { "str": "jury-rigged frame" },
+    "item": "frame",
+    "location": "structure",
+    "standard_symbols": true,
+    "durability": 250,
```

Comment:
Jury rigged frames should actually be a bit tougher than commercial frames, at the significant cost of weight. Later, we'll also need to make them transfer damage to other parts of the car and the occupants, so if you're using a jury rigged vehicle frame you lose the benefit of a crumple zone.

Reply:
The issue with that is that just doing so would violate mass preservation, as somehow a singular frame weights more when jury-rigged
That can be worked around by making dedicated recipes for jury-rigged frames, but that's not quite what I had in mind when making this PR

Reply:
To elaborate, I think having a dedicated recipe for combining, say, 2 heavy duty frames into jury-rigged one *would* make sense, but I don't want to add this to all kinds of frames as this just adds more keypresses for player without really good reason, at least imo
Is there a way for vehicle part to require multiple items, btw? If yes, that would be quite simple solution to that

Reply:
That could be done as a follow up, then; I think the jury-rigged frame *should* be a separate item with higher weight.